Hugo about gcentral (#78)
gcentral/Website: content/about/presentations.md

+++
title = "Presentations about GCentral"
description = "Recorded videos about GCentral and our projects, presented in chronoligical order"
+++

## GDevcon NA 2023 Summer of LabVIEW Squarebattle | Erich Schlieper and Casey May

> Estemed members of the G Idea Exchange, Erich Schlieper and Casey May presents the results of the Summer of LabVIEW, a competition focussing on LabVIEW programming. The competition was started by Derrick Bomaritto and Darren Nattinger. This year the competition focussed on Object Oriented programming through the SquareBattle framework, originally developed by Endigit.

Reviewer: ```suggestion
Esteemed members of the G Idea Exchange, Erich Schlieper and Casey May present the results of the Summer of LabVIEW, a competition focusing on LabVIEW programming. The competition was started by Derrick Bommarito and Darren Nattinger. This year the competition focused on Object Oriented programming through the SquareBattle framework, originally developed by Endigit.
```

{{< youtube 4yYx-dDPBBY >}}

The collection of funding through CauseVox was cancelled in 2023 and the video is no longer matching the current events.

### GDevcon NA 2022 The G-Idea Exchange by GCentral | Bill Welch and Quentin "Q" Alldredge

{{< youtube uu54IJJpUak >}}

## GLASummit 2021

### GLASummit 2021 Quentin "Q" Alldredge presents the GCentral’s G Idea Exchange

Do you have a toolkit that would benefit the community but don't have the resources to finish it? Do you have an awesome idea for tools that would solve problems facing the community but don't know how to bring it about? Introducing GCentral's G Idea Exchange where you can kudo an idea by helping to sponsor it. Help get money to polish off that tool or get someone with the right background to create it and pool our resources to maximize everyone's return on investment.

{{< youtube ugIfFtQrV0c >}}

### GCentral Helping to Write the Web Interface - Chris Cilino, Danielle Jobe, Derrik Bommarito

> GCentralis a community-run initiative that aggregates different package types(VIPackages, NIPackages, GPackages) in one website so that the LabVIEWcommunity can find, share, and collaborate on packages, independent of packagetype. In this presentation, we'll teach how we've created GCentral's websiteand underlying indexing technology so that you can contribute by enhancing oradding new functionality.

Reviewer: ```suggestion
GCentral is a community-run initiative that aggregates different package types (VIPackages, NIPackages, GPackages) in one website so that the LabVIEW community can find, share, and collaborate on packages, independent of package type. In this presentation, we'll teach how we've created GCentral's website and underlying indexing technology so that you can contribute by enhancing or adding new functionality.
```

{{< youtube QHIVSDQ7Y3g >}}

## CLA Summit 2019

> Chris Cillino and Quentin Alldredge presents the initial GCentral proposal at the American CLA Summit in 2019

Reviewer: ```suggestion
Chris Cilino and Quentin Alldredge presents the initial GCentral proposal at the American CLA Summit in 2019
```
